flocker/common/interface.py

```diff
@@ -0,0 +1,40 @@
+# Copyright ClusterHQ Inc.  See LICENSE file for details.
```

Comment:
Should be `_interface.py` and public parts re-exposed in `__init__.py` (because that's an idiom we seem to have adopted... so we should be consistent unless we decide to abandon it entirely).